templates/emails/collective.expense.incomplete.hbs

```diff
@@ -0,0 +1,23 @@
+Subject: Your expense to {{{collective.name}}} for {{currency expense.amount currency=expense.currency}} requires attention
+
+
+{{> header}}
+
+<center>
+  <p>Your expense, <a href="{{config.host.website}}/{{collective.slug}}/expenses/{{expense.id}}">{{expense.description}}</a>, submitted to {{{collective.name}}} was flagged as incomplete and requires your attention:</p>
+  {{#if message}}
+  <p>{{breaklines message}}</p>
```

Comment:
Sounds simpler, safer (for the sanitization) and more consistent with what we do eslsewhere to do:

```suggestion
  <p style="white-space: pre-line;">{{message}}</p>
```

Reply:
That doesn't seem to work everywhere. It doesn't work on maildev, for example.

Reply:
@kewitz Maildev had some breaking changes in their latest versions, we rolled that back. Are the other emails not working as well? I'm surprised because we use this `white-space: pre-line` technique in many places

Reply:
Following up in https://github.com/opencollective/opencollective-api/pull/7531